modules/data_release/ajax/FileUpload.php

```diff
@@ -12,36 +12,37 @@
  * @link     https://github.com/aces/Loris
  */
 
-$DB   = \Database::singleton();
-$user = \User::singleton();
+$DB     = \Database::singleton();
+$user   = \User::singleton();
```

Comment:
Not a big deal but could you update these to use the Factory like the line below?

Reply:
i could... and i wanted to... I just decided to avoid "unrelated changes". I'm gonna do it though

Reply:
Seems related to me! 😄

Reply:
It's nice to leave code cleaner than you found it, especially when it's an [open issue](#4989 ). I think you're safe here

Reply:
sold man stop selling